inst/rmarkdown/templates/outbreak/skeleton/skeleton.Rmd

```diff
@@ -5,25 +5,35 @@ output: github_document
 
 
 # Introduction to this template
-This is a template which can be used to create an automated outbreak situation report. 
+
+This is a template which can be used to create an automated outbreak situation
+report.
 
 - It is organised by time, place and person. 
-- You can type normal text in white spaces (such as here) and r-code in grey spaces (denoted by three backticks and r) (see [Rmarkdown introduction](https://rmarkdown.rstudio.com/articles_intro.html) and [Markdown basics](https://rmarkdown.rstudio.com/authoring_basics.html))
-- Introductions and contents of sections are within square brackets "[...]" and can be deleted as appropriate 
-- Examples of inline code (to automate updating numbers, e.g. line 148), can similarly be removed/updated 
-- Code itself can be deleted, but as a word of caution: make sure you aren't deleting bits where variables are created/manipulated, or at least update them appropriatley
+- You can type normal text in white spaces (such as here) and r-code in grey
+    spaces (denoted by three backticks and r) (see [Rmarkdown
+    introduction](https://rmarkdown.rstudio.com/articles_intro.html) and
+    [Markdown basics](https://rmarkdown.rstudio.com/authoring_basics.html))
+- Introductions and contents of sections are within square brackets "[...]" and
+    can be deleted as appropriate
+- Examples of inline code (to automate updating numbers, e.g. line 148), can
+    similarly be removed/updated
+- Code itself can be deleted, but as a word of caution: make sure you aren't
+    deleting bits where variables are created/manipulated, or at least update
+    them appropriatley
 - ADD LIST ON WHICH PACKAGES WILL BE USED FOR WHAT REASONS! AND HOW TO INSTALL
 - OTHER INFO TO ADD??
 
 
 ```{r setup, include=FALSE}
-knitr::opts_chunk$set(echo = FALSE)
+knitr::opts_chunk$set(echo = FALSE, error = TRUE)
+options(knitr.kable.NA = "-")
```

Comment:
maybe set to "Missing"?

Reply:
The tables would look like this, then:

| Age group (years) |  Deaths | Population | CFR (%) | Lower 95%CI | Upper 95%CI |
| :---------------- | ------: | ---------: | ------: | ----------: | ----------: |
| 0-4               |       1 |          5 |   20.00 |        3.62 |       62.45 |
| 5-9               |       2 |          6 |   33.33 |        9.68 |       70.00 |
| 10-29             |       8 |         27 |   29.63 |       15.85 |       48.48 |
| 30-49             |      14 |         29 |   48.28 |       31.39 |       65.57 |
| 50-79             |       7 |         12 |   58.33 |       31.95 |       80.67 |
| 80+               | Missing |    Missing | Missing |     Missing |     Missing |

cool witchu?

Reply:
Abort. Leave how you had it. Thought it was changing row labels.